src/features/monthly-budget/BudgetForm.js

```diff
@@ -48,8 +50,17 @@ export default function BudgetForm({
     }
   }, [onFormInit]);
 
+  const handleSubmit = (event) => {
+    if (!amountCurrency.value) {
+      return;
+    }
+
+    event.persist();
+    onSubmit(event, amountCurrency.value);
```

Comment:
keep onSubmit [consistency](https://jgefroh.medium.com/why-consistency-is-one-of-the-top-indicators-of-good-code-352ba5d62020), **it should receive all data within a single data structure** or all data spreaded in multiple arguments, but not mixed approaches like this (receiving a single data arbitrarily separated while the rest is attached on the event).

in this commit I did something similar to `handleSubmit`: https://github.com/Mazuh/frifim/pull/5/commits/eee0d8b07d6557aec4b608fd7df1ab09b0b020af